WasatchNET/EEPROM.cs

```diff
@@ -1338,7 +1338,7 @@ public bool read()
                     baudRate = 0;
 
                     hasCooling = false;
-                    hasBattery = true;
+                    hasBattery = true;   // MZ: why?

```

Comment:
What was the reasoning here?

Reply:
I believe I was told at some point these had batteries...I really can't remember, probably can switch it